src/Tizen.NUI/src/public/WindowSystem/QuickPanelClient.cs

```diff
@@ -0,0 +1,441 @@
+
+            res = Interop.QuickPanelClient.GetEventOrientation(ev_info, out state);
+            _tzsh.ErrorCodeThrow(res);
+
+            _orientationChanged?.Invoke(this, (StateOrientation)state);
+        }
+
+        /// <summary>
+        /// Gets the visible state of the quickpanel.
+        /// </summary>
+        /// <returns>The visible state of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public StateVisible GetVisible()
+        {
+            int res;
+            int vis;
+
+            res = Interop.QuickPanelClient.GetVisible(_tzshQpClient, out vis);
+
+            _tzsh.ErrorCodeThrow(res);
+
+            return (StateVisible)vis;
+        }
+
+        /// <summary>
+        /// Gets the window's scrollable state of the quickpanel service window.
+        /// </summary>
+        /// <returns>The scrollable state of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public StateScrollable GetScrollable()
+        {
+            int res;
+            int scroll;
+
+            res = Interop.QuickPanelClient.GetScrollableState(_tzshQpClient, out scroll);
+
+            _tzsh.ErrorCodeThrow(res);
+
+            return (StateScrollable)scroll;
+        }
+
+        /// <summary>
+        /// Sets the window's scrollable state of the quickpanel service window.
+        /// </summary>
+        /// <param name="scroll">The scrollable state.</param>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public void SetScrollable(StateScrollable scroll)
+        {
+            int res;
+
+            res = Interop.QuickPanelClient.SetScrollableState(_tzshQpClient, (int)scroll);
+
+            _tzsh.ErrorCodeThrow(res);
+        }
+
+        /// <summary>
+        /// Gets the orientation of the quickpanel.
+        /// </summary>
+        /// <returns>The orientation of the quickpanel service window.</returns>
+        /// <exception cref="ArgumentException" > Thrown when failed of invalid argument.</exception>
+        /// <exception cref="InvalidOperationException">Thrown when failed because of an invalid operation or no service.</exception>
+        /// <since_tizen> 8 </since_tizen>
+        public StateOrientation GetOrientation()
+        {
```

Comment:
same as above. you can wrap the method with property with making method as internal.

Reply:
I changed this to property named Orientation